### ISSUE TITLE
Calling WebHook does not include `id`

### ISSUE THREAD

**Issue by StevenVerheyen** [mollie/mollie-api-node#149]
Hallo!

I am new to Mollie and tried to implement it with my NodeJS Express server.
In my webshop, I can go to the Mollies payment page (using test mode) and choose the status of the payment. After clicking on `Continue`, my webhook is called. However, the `id` that should in the call is nowhere to be found. I logged the request and response objects just to make sure I did not miss this value.

My webhook url is generated with `ngrok` so I can test this locally. Perhaps this might be wrong?

However.. When I `curl` to the webhook in Terminal, I can see that the webhook works just fine (as I specify the payment id in the curl command).

The payment is listed in the Mollie Dashboard tho.

### Init of Mollie and creating a payment
```javascript
exports.createPayment = asyncHandler(async (req, res, next) => {
  const { value, currency, description, locale, method } = req.body; // these values are correct
  try {
    const mollieClient = createMollieClient({
      apiKey: process.env.PAYMENT_API_KEY,
    });
    mollieClient.payments
      .create({
        amount: {
          value,
          currency,
        },
        description,
        locale,
        method,
        redirectUrl: 'http://localhost:3000',
        webhookUrl: 'https://9c3d7805.ngrok.io/api/v1/payments/webhook', // tunnel broken up after posting this. So this is safe ;)
      })
      .then((payment) => {
        // Forward the customer to the payment.getCheckoutUrl()
        res.status(201).json({
          success: true,
          data: payment,
          checkoutUrl: payment.getCheckoutUrl(),
        });
      })
      .catch((err) => next(err));
  } catch (error) {
    return next(error);
  }
});
```

### Code of webhook, which is a POST request
```javascript
exports.webhookPayment = asyncHandler(async (req, res, next) => {
  const mollieClient = createMollieClient({
    apiKey: process.env.PAYMENT_API_KEY,
  });
  mollieClient.payments
    .get(req.body.id)
    .then((payment) => {
      console.log(payment);
      res.status(200).json({ success: true, data: { payment } });
      if (payment.isPaid()) {
        // Payment is paid, but can be refunded from API / Dashboard at a later point.
        if (
          payment.amountRefunded &&
          parseFloat(payment.amountRefunded.value) > 0
        ) {
          // Mark payment as refunded internally. Send email to consumer regarding successful refund.
          return;
        }

        // Payment has become "paid". Mark payment as paid internally and send order confirmation to consumer. Send consumer to order confirmation page when it reaces your `redirectUrl`
        return;
      }

      if (payment.isAuthorized()) {
        // Payment is authorized. send order confirmation to consumer. Send consumer to order confirmation page when it reaches your `redirectUrl`
        return;
      }

      if (payment.isFailed() || payment.isExpired() || payment.isCanceled()) {
        // Payment is expired / failed / canceled. Make sure the consumer is sent back to checkout page when it reaches your `redirectUrl`.
        return;
      }
    })
    .catch((error) => {
      res.status(200).json({ success: false, data: { payment } });
      // Do some proper error handling.
      // It's recommended to still send a 200 OK, see: https://docs.mollie.com/guides/webhooks#how-to-handle-unknown-ids
    });   
 });
```

I am not sure if you need more info, but if required, please let me know :)
Thanks!

**Comment by RoyBkker** [comment]
I have the same error..

**Comment by StevenVerheyen** [comment]
Hmm is this going to be fixed so that we do not need these packages? I would like to avoid installing 2 additional packages just to get this `id` 😞

**Comment by vernondegoede** [comment]
Thanks @StevenVerheyen and @RoyBkker!

This is not an issue with the Mollie Node API client, but with the way Express parses its `POST` requests.

The easiest way to access the body of such requests is to install the `body-parser` middleware. I noticed that our webhook example is missing this middleware, resulting in an error when this example is copied and the webhook endpoint is triggered. Sorry about that.

I've updated the Express webhook example: https://github.com/mollie/mollie-api-node/blob/master/examples/with-express/02-webhook-verification.js

Hope this helps.
